aws/policy/security-services.yaml

```diff
@@ -15,13 +15,15 @@ Statement:
     Resource:
       - arn:aws:iam::{{ aws_account_id }}:role/ansible-test-sts-*
       - arn:aws:iam::{{ aws_account_id }}:instance-profile/ansible-test-sts-*
-  - Sid: AllowStsAssumeRoleTestsAttachAndDetachRole
+      - arn:aws:iam::{{ aws_account_id }}:role/ansible-test-*
```

Comment:
I'm not sure if it was intentional or not, but do you want to remove the ansible-test-sts-* lines?
`arn:aws:iam::{{ aws_account_id }}:role/ansible-test-sts-*` should be covered by this line. You could also add `- arn:aws:iam::{{ aws_account_id }}:instance-profile/ansible-test-*`. Besides that, looks good to me.

Reply:
I was erring on the side of leaving things very explicit, in case we did decide to break things down differently at some point we wouldn't overlook the different prefixes in use. I don't feel especially strongly either way so whatever seems sanest to others who need to work with the policies works for me.

Reply:
I think they should be removed, but for the time being until we sort out the tests, looks good to me.